root/layout/index.js

```diff
@@ -119,32 +136,49 @@ const Layout = ({$c, ...props}) => (
         </div>}
 
       {!!($c.stash.new_edit_notes &&
-          $c.stash.new_edit_notes_mtime > getRequestCookie($c.req, 'new_edit_notes_dismissed_mtime', 0) &&
-          ($c.user.is_limited || getRequestCookie($c.req, 'alert_new_edit_notes', 'true') !== 'false')) &&
+          $c.stash.new_edit_notes_mtime > getRequestCookie(
+            $c.req, 'new_edit_notes_dismissed_mtime', 0,
+          ) &&
+          ($c.user.is_limited || getRequestCookie(
+            $c.req, 'alert_new_edit_notes', 'true',
```

Comment:
Same as above

Reply:
Needs to be 

```
          ($c.user.is_limited ||
          getRequestCookie($c.req, 'alert_new_edit_notes', 'true') !==
          'false')) &&
```

to fit the lines, but guess that'd still be better.